packages/e2e-test-utils-playwright/src/request-utils/pages.ts

```diff
@@ -74,7 +74,7 @@ export async function createPage(
 	const page = await this.rest< Page >( {
 		method: 'POST',
 		path: `/wp/v2/pages`,
-		params: payload,
+		data: { ...payload },
```

Comment:
The `createPage` sent data via query arguments for the `POST` request; it should be sent a body.